.eslintcache

```diff
@@ -0,0 +1 @@
ages":"84","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"50"},{"filePath":"85","messages":"86","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"50"},{"filePath":"87","messages":"88","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"50"},{"filePath":"89","messages":"90","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"50"},{"filePath":"91","messages":"92","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"50"},{"filePath":"93","messages":"94","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"50"},"/home/braifz/Workplace/ReactJS/coso-v0.2/src/index.js",[],["95","96","97","98","99","100","101"],"/home/braifz/Workplace/ReactJS/coso-v0.2/src/components/App.js",[],"/home/braifz/Workplace/ReactJS/coso-v0.2/src/components/ModalSwitch.js",[],"/home/braifz/Workplace/ReactJS/coso-v0.2/src/components/Layout.jsx",[],"/home/braifz/Workplace/ReactJS/coso-v0.2/src/components/ViewRecipe.jsx",[],"/home/braifz/Workplace/ReactJS/coso-v0.2/src/Pages/Profile.jsx",[],"/home/braifz/Workplace/ReactJS/coso-v0.2/src/Pages/Home.jsx",[],"/home/braifz/Workplace/ReactJS/coso-v0.2/src/Pages/About.jsx",[],"/home/braifz/Workplace/ReactJS/coso-v0.2/src/Pages/SearchResultsPage.jsx",[],"/home/braifz/Workplace/ReactJS/coso-v0.2/src/Pages/SignIn.jsx",[],"/home/braifz/Workplace/ReactJS/coso-v0.2/src/Pages/SignUp.jsx",[],"/home/braifz/Workplace/ReactJS/coso-v0.2/src/components/Footer.jsx",[],"/home/braifz/Workplace/ReactJS/coso-v0.2/src/components/Header.jsx",[],"/home/braifz/Workplace/ReactJS/coso-v0.2/src/components/HomeScreen.jsx",[],"/home/braifz/Workplace/ReactJS/coso-v0.2/src/components/SearchResults.jsx",[],"/home/braifz/Workplace/ReactJS/coso-v0.2/src/components/ItemRecipe.jsx",[],"/home/braifz/Workplace/ReactJS/coso-v0.2/src/components/SearchFieldCollapse.jsx",[],"/home/braifz/Workplace/ReactJS/coso-v0.2/src/components/CardList.jsx",[],"/home/braifz/Workplace/ReactJS/coso-v0.2/src/components/NutritionalTable.jsx",[],"/home/braifz/Workplace/ReactJS/coso-v0.2/src/components/RadioInput.jsx",[],"/home/braifz/Workplace/ReactJS/coso-v0.2/src/components/CardItem.jsx",[],"/home/braifz/Workplace/ReactJS/coso-v0.2/src/components/RangesInput.jsx",[],"/home/braifz/Workplace/ReactJS/coso-v0.2/src/components/CheckboxInput.jsx",[],{"ruleId":"102","replacedBy":"103"},{"ruleId":"104","replacedBy":"105"},{"ruleId":"106","replacedBy":"107"},{"ruleId":"108","replacedBy":"109"},{"ruleId":"110","replacedBy":"111"},{"ruleId":"112","replacedBy":"113"},{"ruleId":"114","replacedBy":"115"},"jsx-a11y/accessible-emoji",[],"jsx-a11y/label-has-for",[],"lines-around-directive",["116"],"global-require",[],"no-buffer-constructor",[],"no-new-require",[],"no-path-concat",[],"padding-line-between-statements"]
```

Comment:
.eslintcache is already in .gitignore for this branch, I have no idea why this is still getting tracked.  @tonoezep any ideas?

Reply:
yes, Braifz has started this branch before this was added to gitignore, then merge it with the master's HEAD
just delete it, and start all new branches from the latest version of master ⭐